src/pages/home/components/LoginBanner.tsx

```
@@ -46,7 +77,9 @@ const LoginBanner = () => {
           </span>
 
           <div className="mt-[10px] text-white font-T01-B">
-            {todoData && todoData.jobName ? `${todoData.jobName} 시작하는 중` : '직업 준비중'}
+            {todoData && todoData.jobName
+              ? `${todoData.jobName} 시작하는 중`
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)